run.go

```diff
@@ -121,9 +124,20 @@ func updateConfigWithFlags(rootCmd *cobra.Command, cnfg *config.Config, programA
 	_ = subCmd.ParseFlags(flags)
 	// initialize config from file
 	err := config.ReadAndMergeWithFlags(globalFlagValues, cnfg)
+	setDefaultCoordinator(cnfg.Logger)
 	return err
 }
 
+func setDefaultCoordinator(logger *log.Logger) {
```

Comment:
Why does this function get a logger as parameter?

Reply:
It logs if it can not assign the default URL. Do you suggest an alternative flow?

Reply:
Maybe you can just return the error. Since if the coordinator URL cannot be set we shouldn't continue to run the command anyway.

Reply:
I think for many usecases this may be non-critical. For those cases just logging makes sense for me. Wdyt?

Reply:
I think it's a serious problem if the user intends to connect to Viridan but it can't since the coordinator URL was not set correctly. What we can do is to call `setDefaultCoordinator` only when we know the user wants to connect to the cloud.

Reply:
Good idea, updated as discussed https://github.com/hazelcast/hazelcast-commandline-client/pull/145/commits/e51609e62016f69e0a9e8fbe2c7e3617f471846d